docs/api.md

```diff
@@ -53,13 +53,79 @@ In our experience, any system that is successful needs to grow and change as new
 
 What constitutes a compatible change and how to change the API are detailed by the [API change document](devel/api_changes.md).
 
+
+## API groups
+
+To make it easier to extend the Kubernetes API, we are in the process of implementing *API groups*.
+The API group is specified in a REST path and in the `apiVersion` field of a serialized object.
+
+Currently there are two API groups in use:
+
+1. the "core" group, which is at REST path `/api/v1` and is not specified as part of the `apiVersion` field, e.g.
+   `apiVersion: v1`.
+1. the "experimental" group, which is at REST path `/apis/experimental/$VERSION`, and which uses
+  `apiVersion: experimental/$VERSION`, where currently `apiVersion: experimental/v1alpha1`.
+  The name `experimental` implies that API groups themselves are currently under development.  The actual
```

Comment:
These two sentences are still very confusing to me.  Needs further wordsmithing, IMO.  Will come back with some suggestions once I've grokked the rest of the doc.